src/components/VList/VListGroup.js

```diff
@@ -107,6 +107,13 @@ export default {
 
       this.isActive = !this.isActive
     },
+    onKeyDown: function onKeyDown (e) {
```

Comment:
Use method shorthand, and should be called `onKeydown`

Reply:
done